packaging/build_deb

```diff
@@ -9,14 +9,13 @@
 set -ex
 
 readonly SRCDIR="${PWD}"
-readonly TARGET_DIR="${1}"
-readonly WORKDIR="$(mktemp -d --tmpdir=/tmp build_deb.XXXXXX)"
+readonly TARGET_DIR="${SRCDIR}/$1"
```

Comment:
Optional: `DESTDIR`

I think I saw `DESTDIR` or `OUTDIR` more often than `TARGET_DIR`; it's a matter of taste though.

Reply:
Renamed to `DESTDIR`.